RestaurantManager/Pages/Employees/EmployeeDetailViewModel.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.IO;
+using System.Net;
+using Prism.Common;
+using Prism.Navigation;
+using RestaurantManager.Model;
+using RestaurantManager.Services.Network;
+using Xamarin.Forms;
+using XCT.Popups.Prism;
+
+namespace RestaurantManager.Pages.Employees
+{
+    public class EmployeeDetailViewModel : ViewModelBase
+    {
+        private User _employee;
+        private ImageSource _imageSource;
+        public ImageSource ImageSource
+        {
+            get => _imageSource;
+            set => SetProperty(ref _imageSource, value);
+        }
+        public User Employee
+        {
+            get => _employee;
+            set => SetProperty(ref _employee, value);
+        }
+        public EmployeeDetailViewModel(INavigationService navigationService, IPopupService popupService, INetworkService networkService) : base(navigationService, popupService, networkService)
+        {
+            Title = "Employee Detail";
+        }
+
+        public override void OnNavigatedTo(INavigationParameters parameters)
+        {
+            base.OnNavigatedTo(parameters);
+            var webClient = new WebClient();
+            User employee;
+            parameters.TryGetValue("employee", out employee);
+            Employee = employee;
+
+            try
+            {
+                var imgBytes = webClient.DownloadData(
```

Comment:
check if there is network before performing the action
display error the error if you catch it